scripts/functions.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#  Set date command based on OS
+date_cmd () {
```

Comment:
how is this working?

it's invoked at `date_command` but I can't see anything calling `date_cmd`

Reply:
yes you're correct. removed the uncalled function, changed functions.sh to common.sh. changed all references in dependant scripts to source common.sh and call $date_command. tested all scripts call the date command as expected.